### ISSUE TITLE
Import corpus fails...

### ISSUE THREAD

**Issue by kchall** [PhonologicalCorpusTools/CorpusTools#428]
So, I think the logging is working now (#419), but no new corpus actually gets created! Yikes! No error message, and it looks as though it is processing, but then nothing appears in the corpus menu, and a search of my computer yields no results for the new corpus.

**Comment by kchall** [comment]
Now I'm super confused. I just imported a different corpus, the Lexique corpus, and this time the import worked, but there's not a new / overwritten log file (i.e., it still has the log file info. for the Lezgian corpus, which didn't actually import).

**Comment by mmcauliffe** [comment]
I can't reproduce that, the Lezgian corpus imports fine for me.  The log should only overwrite itself when the program starts up, so it should have the Lezgian plus Lexique info.  It may be an issue that you have it open in an editor that exerts a lock on the file, in which case, PCT can't write to it.

**Comment by kchall** [comment]
Ah, I see...you're right, the logging info. is there after all, for all the corpora. But the Lezgian corpus is not loading. I tried re-starting PCT, but still nothing. This is coming from the running-text version, called Lezgian_corpus_2.txt (https://www.dropbox.com/s/fqnlz5tputzwng2/Lezgian_corpus_2.txt?dl=0).

Here is the log file for it:

INFO:root:Importing transcription corpus named Lezgian_corpus_2_TEST_AGAIN
INFO:root:Path: 
INFO:root:Annotation type info
INFO:root:--------------------
INFO:root:
INFO:root:transcription:
    Ignored characters: !, (, )
    Digraphs: tʷ', tʷʰ, tʷH, tʷ, sʷ, zʷ, qʷH, kʷH, kH, tH, pH, qH, tsʰ, tsH, tʃH, gʷ, kʷʰ, tsʷʰ, tsʷH, tsʷ', tsʷ, ts', ts, t', kʷ', k', p', tʃʰ, tʃ', tʃ, qʷʰ, qʷ', q', qʷ, kʷ, pʰ, tʰ, kʰ, qʰ, ʁʷ, χʷ
    Transcription delimiter: None
    Morpheme delimiters: -
    Number behavior: None

**Comment by mmcauliffe** [comment]
Hmm, it works for me.  I'll try it on a mac.

**Comment by mmcauliffe** [comment]
It works on the Speech in Context mac.  I'm running out of ideas...

**Comment by kchall** [comment]
That is truly bizarre. I have no ideas, either... I mean, the log file is there, it looks like it's processing; I know how to load a corpus, and it certainly works with some other corpora (e.g., the column-delimited Lexique corpus, the test_text_transcription_morpheme_boundaries test file, and the Fox in Sox running text). 

But, I also have the same problem with the Gitksan running text corpus -- creates a log file no problem, but no new corpus. (https://www.dropbox.com/s/9h75850r4o5tb1l/Gitksan_corpus_orth.txt?dl=0)

So, languages and corpora we actually care about seem to not be working...? Sigh...

**Comment by kchall** [comment]
Ok, I have no clue. It works on my other laptop. I have tried restarting my whole computer, re-syncing, and re-loading the corpus several times. It is absolutely nowhere, even though the log file is there and everything seems hunky dory. I'm going to assume this is a weird issue on my computer, though it does worry me.

Hmm. I tried letting it overwrite my existing version of the corpus without changing the name (but changing the parsing settings so that I could determine if it was a new version), and that DOES seem to have worked. Don't know if that gives us any clues?
